src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,376 @@
+# Microscopy
+
+Support for Microscopy was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+<!---
+The following link is temporary and points to the bep031 branch of bids-examples,
+update to 'https://github.com/bids-standard/bids-examples' when examples are merged.
+-->
+Microscopy datasets formatted using this specification are available on the
+[BIDS examples repository](https://github.com/neuropoly/bids-examples/tree/bep031)
+(`microscopy_SEM001` and `microscopy_SPIM001`) and can be used for practical
+guidance when curating a new dataset.
+
+Further Microscopy datasets are available:
+
+-   In PNG format:  [`data_axondeepseg_sem`](https://doi.org/10.5281/zenodo.5498378)
+-   In OME-TIFF format: (link to the dataset - pending)
+
+## Microscopy recording data
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+Template:
+```Text
+sub-<label>/
+    [ses-<label>/]
+        microscopy/
```

Comment:
We haven’t found a good abbreviation for microscopy, so we decided to keep it as is. We are not comfortable with `micro` alone because it has another meaning on its own as a unit prefix.

Reply:
Looking up [abbreviations for microscopy](https://www.google.com/search?q=abbreviations+for+microscopy) I see some journals use "microsc". Only saving 3 characters there, but possibly less ambiguous than `micro` or `micr`.

Despite posting twice on the matter, I really don't have a strong feeling here. Just trying to offer a suggestion.

cc @yarikoptic to drag the discussion over here.

Reply:
i'm happy with `micr`

Reply:
Thank you everyone for the feedback, it seems that `micr` would be the best choice, so I'll make the change in the spec and in our examples.